content/docs/getting-started/cms-set-up.md

```diff
@@ -2,7 +2,16 @@
 title: Set Up the CMS
 prev: /docs/getting-started/introduction
 next: /docs/getting-started/edit-content
+last_edited: '2020-08-18T07:32:54.314Z'
 ---
+## Clone the demo
+
+To get started, clone the [demo](https://github.com/tinacms/tina-intro-tutorial), install its dependencies, and start the dev server.
+
+    git clone git@github.com:tinacms/tina-intro-tutorial.git my-tina-app
+    cd my-tina-app
+    yarn install
+    yarn start
 
 In this step we will install `tinacms`, create a CMS instance, wrap our app in the `TinaProvider`, and configure the CMS options.
```

Comment:
It feels awkward that the 'step intro sentence' seems like a part of the 'clone the demo' header.